var/spack/repos/builtin/packages/intel-daal/package.py

```diff
@@ -48,11 +41,12 @@ class IntelDaal(IntelInstaller):
 
     provides('daal')
 
-    def install(self, spec, prefix):
-
-        self.intel_prefix = os.path.join(prefix, "pkg")
```

Comment:
@adamjstewart I don't recall putting that in there and am not sure why it is in there. Of course it is possible that I did put it in there and forgot...